benchmark/report/script/report_processor.py

```diff
@@ -12,28 +13,41 @@
 ARROW_DOWN = "\U00002198"
 ARROW_EQ = "\U00002194"
 
-
-def process(dir):
+def process(reports_dir, benchmark_dir):
     print("**Automated k6 Benchmark Report**")
     print("--------------------------------")
     report_list = sorted([
-        report for report in os.listdir(dir) if os.path.isfile(get_report_uri(dir, report))
+        report for report in os.listdir(reports_dir) if os.path.isfile(get_report_uri(reports_dir, report))
     ], reverse=True)
 
-    last_report = json.load(open(get_report_uri(dir, report_list.pop(0))))["data"]
+    if len(report_list) == 0:
+        raise Exception("Can't compile the summary - there are no reports to compare with. " +
```

Comment:
Do we want to have an error here? I'm not sure. It's CI and ultimately it's not Build's fault, that you don't have any prior tags pushed.